frontend/src/main.tsx

```diff
@@ -1,16 +1,15 @@
 import { StrictMode } from "react";
 import ReactDOM from "react-dom/client";
 import { RouterProvider, createRouter } from "@tanstack/react-router";
-
 import clientMetadata from "../client-metadata";
-
 // Import the generated route tree
 import { routeTree } from "./routeTree.gen";
-
 import reportWebVitals from "./reportWebVitals.ts";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { BrowserOAuthClient } from "@atproto/oauth-client-browser";
 
+export const habitatProxy = "did:web:habitat.network#habitat"
```

Comment:
this should be in a separate file otherwise it's a circular dependency. not really an issue in typescript but better to avoid